app/mailers/rrd_new_disability_claim_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class RrdNewDisabilityClaimMailer < ApplicationMailer
+  def build(submission, health_data)
+    @id = submission.id
+    @submitted_claim_id = submission.submitted_claim_id
+    @created_at = submission.created_at
+    @created_at = submission.created_at
+    @disabilities = submission.disabilities
+    @bp_readings = health_data[:bp_readings]
+    @medications = health_data[:medications]
+    @multi_issue = submission.disabilities.count > 1
+    @single_issue = submission.disabilities.count == 1
+
+    template = File.read('app/mailers/views/rrd_new_disability_claim_mailer.html.erb')
+
+    mail(
+      to: Settings.rrd.pact_tracking.recipients,
+      subject: "MA claim - #{submission.submitted_claim_id}",
```

Comment:
Do we want to keep this more general? This class has no other mention to MAS.
```suggestion
      subject: "NEW claim - #{submission.submitted_claim_id}",
```